scripts/fonts.js

```diff
@@ -0,0 +1,143 @@
+// read font file
+import { Font } from "fonteditor-core";
+import fs from "fs";
+import { execSync } from "child_process";
+import request from "sync-request";
+
+function createRangeArray(start, end) {
+  return Array.from({ length: end - start + 1 }, (_, i) => start + i);
+}
+
+function loadGoogleFonts(fontSpec, destFolder) {
+  for (const fontFamily in fontSpec) {
+    const downloadLockFile = `${downloadLockFolder}/${fontFamily}.lock`;
+    if (fs.existsSync(downloadLockFile)) {
+      console.log(`Already downloaded ${fontFamily}`);
+      continue;
+    }
+    const variants = fontSpec[fontFamily].join(",");
+    execSync(
+      //Requires google-font-installer
+      `npm exec -- gfi download "${fontFamily}" -v ${variants} -d ${destFolder}`,
+      (error, stdout, stderr) => {
+        if (error) {
+          console.log(`error: ${error.message}`);
+          console.log(
+            `Is google-font-installer present? Install with "npm install google-font-installer`
```

Comment:
This is sort of superfluous in that we should always assume that the dependencies were installed.

Reply:
Removed in 7d5a2f6